client/app/hosting/ssl/details/hosting-ssl-details.controller.js

```diff
@@ -0,0 +1,43 @@
+angular
+    .module("App")
+    .controller("hostingSSLDetailsController", class HostingSSLDetailsController {
+        constructor (constants, Hosting, $scope, translator) {
+            this.constants = constants;
+            this.Hosting = Hosting;
+            this.$scope = $scope;
+            this.translator = translator;
+        }
+
+        $onInit () {
+            this.hosting = this.$scope.currentActionData;
+            this.comodoSupportLink = this.constants.urls.comodoSupport;
+
+            this.retrievingDetails();
+        }
+
+        retrievingDetails () {
+            this.isLoading = true;
+
+            return this.Hosting
+                .retrievingDetails(this.hosting.serviceName)
+                .then((sslReport) => {
+                    const sslReportEntries = Object.entries(sslReport);
```

Comment:
https://lodash.com/docs/3.10.1#pairs instead?

Reply:
Good one !

Reply:
I'm actually doing the opposite of this function : I'm turning a two dimensional array into an array of objects. I could remove altogether if this feels strange to you, I felt like `sslReportEntry.key` was more readable than `sslReportEntry[0]`